hazelcast/src/main/java/com/hazelcast/nio/tcp/SocketAcceptorThread.java

```diff
@@ -47,8 +49,20 @@
     private final SwCounter eventCount = newSwCounter();
     @Probe
     private final SwCounter exceptionCount = newSwCounter();
+    // count number of times the selector was recreated (if selectWorkaround is enabled)
+    @Probe
+    private final SwCounter selectorRecreateCount = newSwCounter();
+    // last time select returned
     private volatile long lastSelectTimeMs;
 
+    // When true, enables workaround for bug occuring when SelectorImpl.select returns immediately
```

Comment:
This is now logged on acceptor thread startup: https://github.com/vbekiaris/hazelcast/blob/9e10fffc8d5a3b4c54ad506816d9cfb138df7de8/hazelcast/src/main/java/com/hazelcast/nio/tcp/SocketAcceptorThread.java#L96